docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+
+services:
+
+  consul-server-1:
+    image: consul:latest
+    networks:
+      - trento
+    ports:
+      - "8400:8400"
+      - "8500:8500"
+      - "8600:8600"
+      - "8600:8600/udp"
+    command: "agent -server -bootstrap-expect 3 -ui -client 0.0.0.0"
+
+  consul-server-2:
+    image: consul:latest
+    networks:
+      - trento
+    command: "agent -server -retry-join consul-server-1 -client 0.0.0.0"
+
+  consul-server-3:
+    image: consul:latest
+    networks:
+      - trento
+    command: "agent -server -retry-join consul-server-1 -client 0.0.0.0"
+
+  trento-web:
+    build: .
+    networks:
+      - trento
+    ports:
+      - "8080:8080"
+    environment:
+      - CONSUL_HTTP_ADDR=consul-server-1:8500
+    command: "/app/trento web serve -p 8080"
```

Comment:
I think that in order to have the syntax, this last `command` should be `web serve -p 8080`, where `/app/trento` is already defined in the trento dockerfile

Reply:
You make a good point, and you made me realise another change that we need to make!
Having `command: web serve -p 8080` would only work if `/app/trento` is set as `ENTRYPOINT` in the Dockerfile, instead of it being a `CMD`. So basically we should swap `CMD` with `ENTRYPOINT` there, and leave `CMD` empty.
This would result in `docker run trento` running the default application command (which is `help`, I think), which is exactly what users would expect!

Reply:
Yes, indeed it's cleaner to use ENTRYPOINT in the Dockerfile and only the arguments in the `command` part of the compose. Thanks!